### PR TITLE
moving GetServerURL, ExportToXLSX, GetInstances, CVSSCalculator, Repl…

### PR DESCRIPTION
**Pull request by maimorag**
…aceMatchGroup, BreachConfirmationHTML to CommonScripts

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress

## Related Issues
N/A

## Description
moving GetServerURL, ExportToXLSX, GetInstances, CVSSCalculator, ReplaceMatchGroup, BreachConfirmationHTML to CommonScripts

## Minimum version of Cortex XSOAR
- [X] 6.0.0

## Does it break backward compatibility?
   - [X] Yes

### CODE REVIEW

File: Packs/CVSS/ReleaseNotes/1_0_3.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"breakingChanges":true,"breakingChangesNotes":"The script GetInstances moved to *CommonScripts* pack. Make sure to update the *CommonScripts* pack to the latest version in order to use the script."}
```
Comment (Shellyber) [inline Packs/CVSS/ReleaseNotes/1_0_3.json:1]: Wrong script name - please go over all the rn and verify its the right script

File: Packs/ModulesManagement/ReleaseNotes/1_0_4.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"breakingChanges":true,"breakingChangesNotes":"The script CVSSCalculator moved to *CommonScripts* pack. Make sure to update the *CommonScripts* pack to the latest version in order to use the script."}
```
Comment (Shellyber) [inline Packs/ModulesManagement/ReleaseNotes/1_0_4.json:1]: again wrong script name

File: Packs/ReplaceMatchGroup/.pack-ignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[known_words]
```
Comment (Shellyber) [inline Packs/ReplaceMatchGroup/.pack-ignore:1]: remove this

File: Packs/CVSS/ReleaseNotes/1_0_3.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#### CommonScripts
```
Comment (Shellyber) [inline Packs/CVSS/ReleaseNotes/1_0_3.md:1]: The header is wrong in all placed, please follow the right format

**Review by Shellyber — CHANGES_REQUESTED** [review]
nice

File: Packs/ModulesManagement/ReleaseNotes/1_0_4.json (hunk 2 above)
Comment (maimorag) [reply Packs/ModulesManagement/ReleaseNotes/1_0_4.json:1]: done

File: Packs/CVSS/pack_metadata.json [hunk 5]
```diff
@@ -1,8 +1,8 @@
 {
-    "name": "CVSS",
-    "description": "Tools associated with CVSS scores and calculations.",
+    "name": "CVSS  (Deprecated)“",
+    "description": "Deprecated. Use from CommonScripts instead.",
```
Comment (Shellyber) [inline Packs/CVSS/pack_metadata.json:3]: What about hidden: true ? with all the deprecated  packs

Comment (maimorag) [reply Packs/CVSS/pack_metadata.json:3]: oh no

**Review by Shellyber — APPROVED** [review]

**Pull request merged**